src/osemgrep/configuring/Semgrep_settings.mli

```diff
@@ -0,0 +1,8 @@
+type t = {
```

Comment:
Maybe add a comment like
(* content of the the ~/.semgrep/settings.yml file. See also Semgrep_envvars.user_settings_file *)